tests/Extension/SymfonyTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the phpunit-json-assertions package.
+ *
+ * (c) Enrico Stahn <enrico.stahn@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace EnricoStahn\JsonAssert\Tests\Extension;
+
+use EnricoStahn\JsonAssert\Extension\Symfony;
+use EnricoStahn\JsonAssert\Tests\Utils;
+use Symfony\Component\HttpFoundation\Response;
+
+class SymfonyTest extends \PHPUnit_Framework_TestCase
+{
+    public function testAssertJsonMatchesSchema()
+    {
+        $schema = Utils::getSchemaPath('test.schema.json');
+        $response = new Response(file_get_contents(Utils::getJsonPath('simple.json')));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\EnricoStahn\JsonAssert\Tests\Utils' in method 'testAssertJsonMatchesSchema'.](https://www.codacy.com/app/estahn/phpunit-json-assertions/file/2256991341/issues/source?bid=3181605&fileBranchId=3182397#l23)